editor/src/core/workers/parser-printer/ts-creator/index.ts

```diff
@@ -32,7 +32,7 @@ export default function create(code: string): string {
     ScriptKind.TSX,
   )
 
-  const factoryFile = transformSourceFile(file)
+  const factoryFile = transformSourceFile(code, file)
```

Comment:
What was the reason for this change?

Reply:
Need the original text to get the comments from it.